apps/lib/apps.c

```diff
@@ -942,13 +945,14 @@ int load_key_certs_crls(const char *uri, int format, int maybe_stdin,
     OSSL_PARAM itp[2];
     const OSSL_PARAM *params = NULL;
 
+    /* 'failed' describes type of credential to load for potential error msg */
     if (failed == NULL) {
         if (!quiet)
-            BIO_printf(bio_err, "Internal error: nothing to load from %s\n",
+            BIO_printf(bio_err, "Internal error: nothing was requested to load from %s\n",
                        uri != NULL ? uri : "<stdin>");
         return 0;
     }
-    ERR_set_mark();
+    ERR_set_mark(); /* suppress (most) low-level errors during loading */
 
     SET_EXPECT1(ppkey, OSSL_STORE_INFO_PKEY);
     SET_EXPECT1(ppubkey, OSSL_STORE_INFO_PUBKEY);
```

Comment:
Don't these consecutive `SET_EXPECT1` calls imply that `expect == 0` when all's said and done?  Would it not be simpler to just initialise all pointers to NULL and set `expect = 0`?
And then, aren't many of the `SET_EXPECT` calls below redundant?

Reply:
Ugh, after long time (I coded this in early 2022), 
hard to dig again into those details  I currently cannot afford doing so.

Anyway the scope of this PR is just a minor refactoring for better readability - these lines were not touched.